src/components/styles/ImagePreview.css

```diff
@@ -1,24 +1,48 @@
+@import "../../pages/app/styles/GlobalStyles.css";
+/* 
+*Tien edited with global styles
+*3rd October 2020
+*/
 .avatar-box{
-    border:2px solid black;
-    border-radius: 5px;
-    height: 200px;
+    border:var(--global-border-color);
+    border-radius:var(--global-border-radius);
+    height: 300px;
     width:335px;
     margin-top:20px;
-    margin-bottom:20px;
-    background-color: rgb(22, 206, 181);
+    margin-bottom: -20px
```

Comment:
Why is it -20px? @TrNgTien

Reply:
make it closer with the element above @kenpham4real

Reply:
oops 2 margin buttom hmmm let's me check and delete that

Reply:
To make it closer to the element above, please use the proper one: **margin-top**.

Also, another code **convention**:
As this is a global component, we'll use it in many places, relevant styling should be coded in the parent component instead of the children one.

If you need to make it closer to... or anything like that, please style it in the parent component instead.
This'll make it easier for debugging and avoid any conflicts in styling from other parent component wanting to use the children

Reply:
ok got it, so i will fix it and push it again? @kenpham4real

Reply:
Would love to. Many thanks

Reply:
ok i will fix it asap and push it again